httpServer/httpServer.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"time"
+)
+
+func getDateTime(w http.ResponseWriter, req *http.Request) {
+
+	currentTime := time.Now()
+	response := currentTime
+	json.NewEncoder(w).Encode(response)
```

Comment:
Instead of initializing a new variable just encode currentTime directly
```suggestion
	json.NewEncoder(w).Encode(currentTime)
```